node/pkg/driver/node_utils.go

```diff
@@ -230,11 +238,37 @@ func (n NodeUtils) ParseFCPorts() ([]string, error) {
 	return fcPorts, nil
 }
 
-func (n NodeUtils) Exists(path string) bool {
+func (n NodeUtils) IsPathExists(path string) bool {
 	_, err := os.Stat(path)
 	if err != nil {
+		if !os.IsNotExist(err) {
+			return false
+			logger.Warningf("Check is file %s exists returned error %s", path, err.Error())
+		}
 		return false
 	}
 
 	return true
 }
+
+func (n NodeUtils) IsDirectory(path string) bool {
+	targetFile, err := os.Stat(path)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			logger.Warningf("Check is directory %s returned error %s", path, err.Error())
+		}
+		return false
+	}
+	return targetFile.Mode().IsDir()
+}
+
+// Deletes file or directory with all subdirectory and files
+func (n NodeUtils) RemoveFileOrDirectory(path string) error {
+	return os.RemoveAll(path)
+}
+
+// path: file/dir path
+// return: path to the file/dir if accessed from pod
+func GetPodPath(filepath string) string {
```

Comment:
https://github.com/IBM/ibm-block-csi-driver/pull/117#discussion_r373863511
...and parameter name to path

Reply:
if you disagree, write. do not just secretly resolve.